Formula/jet.rb

```diff
@@ -0,0 +1,19 @@
+class Jet < Formula
+  desc "Type safe SQL builder with code generation and auto query result data mapping"
+  homepage "https://github.com/go-jet/jet"
+  url "https://github.com/go-jet/jet/archive/refs/tags/v2.10.0.tar.gz"
+  sha256 "6345ed0ea92159ff2648cc1b4154478da8a2b16a61d47c74e394169897ba8130"
+  license "Apache-2.0"
+  head "https://github.com/indentapis/access.git", branch: "main"
```

Comment:
Perhaps you forgot to change this line 👀

Reply:
😬